src/main/java/com/ecommerce/ECommerceApp/model/Order.java

```diff
@@ -1,13 +1,17 @@
 package com.ecommerce.ECommerceApp.model;
 
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+
 @Component
 public class Order{
     private int orderID;
-    private dateTime orderDate;
+    private LocalDateTime orderDate;
     private int userID;
     private float totalAmount;
 
-    public Order(int orderID, dateTime orderDate, int userID, float totalAmount) {
+    public Order(int orderID, LocalDateTime orderDate, int userID, float totalAmount) {
```

Comment:
Implementation of LocalDateTime over dateTime to help reduce future errors and ensure dates are handled consistently throughout classes. Overall reducing incurred technical debt.

Source: https://martinfowler.com/bliki/TechnicalDebt.html